### ISSUE TITLE
Tv media box from telenet uses new version of play.

### ISSUE THREAD

**Issue by Bosken85** [add-ons/plugin.video.yelo#16]
First of all, much gratitude for investing the time to make this. I was wondering, are there any plans of supporting the new yelo play website from telenet? I have their new media box which prevents me from using yelo play. They always redirect me to their new online play portal.

**Comment by shycoderX** [comment]
Is this the new Yelo version only certain subscriptions can see? If you want me to take a look at this then please provide a Fiddler / Chrome Trace

**Comment by Bosken85** [comment]
It is indeed telenet's new version of their online tv. The old one is hosted under https://www.yeloplay.be/ but the new one is hosted on https://www.telenettv.be . People like me with their latest streaming box are automatically switched to this new platform and are prevented from using the old api. ( lame! :) ) I will see how i can go about providing this trace you need.

**Comment by shycoderX** [comment]
> It is indeed telenet's new version of their online tv. The old one is hosted under https://www.yeloplay.be/ but the new one is hosted on https://www.telenettv.be . People like me with their latest streaming box are automatically switched to this new platform and are prevented from using the old api. ( lame! :) ) I will see how i can go about providing this trace you need.

Yes, I would take a look but without a trace I can't reverse engineer the API calls, and with my current subscription I can't access the new site. I have only access to the old site.

**Comment by fripsy** [comment]
Same here, I don't have access (yet) to the new site either...

**Comment by LuccaBorsellino** [comment]
> 
> 
> > It is indeed telenet's new version of their online tv. The old one is hosted under https://www.yeloplay.be/ but the new one is hosted on https://www.telenettv.be . People like me with their latest streaming box are automatically switched to this new platform and are prevented from using the old api. ( lame! :) ) I will see how i can go about providing this trace you need.
> 
> Yes, I would take a look but without a trace I can't reverse engineer the API calls, and with my current subscription I can't access the new site. I have only access to the old site.



> 
> 
> Same here, I don't have access (yet) to the new site either...

I can supply you an account if you want to
